tests/test_app/TestApp/ServiceProvider/PersonServiceProvider.php

```diff
@@ -19,4 +19,14 @@ public function services(ContainerInterface $container): void
     {
         $container->add('sally', json_decode('{"name":"sally"}'));
     }
+
+    public function provides(string $id): bool
```

Comment:
Is this a new method that service providers will need to implement? Should we provide a default implementation that reads the property so that application developers don't need to update their service providers immediately?

Reply:
Yup it's a new method on the interface, I will into a default implementation.

Reply:
@markstory Done.